docs/pages/distribution/runtime-version.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Runtime Version
+---
+
+Expo Update allows you to publish over the air updates to apps. The update must be for an app running a
+[compatible](../workflow/publishing/#what-version-of-the-app-will-my) runtime. You may use a runtime other than an Expo SDK, by specifying a runtime version.
```

Comment:
Are we updating this section too?

Reply:
yea once the "main" description stabilizes I will go and look at all the references.